### ISSUE TITLE
Include duplicate reads for pileup

### ISSUE THREAD

**Issue by andurill** [alimanfoo/pysamstats#91]
Hi @alimanfoo,

Thanks so much for developing this package! I used it recently to get base level pileup information and noticed that its not counting duplicate reads for pileup. I tried getting the pileup data by both running pysamstats from command line and calling the 'load_variation_strand' method from within python, as show below. Neither of these approaches included the duplicate reads. If I'm not wrong, there is no option to set the no_dup flag to true/false when calling the 'load_variation_strand' method. However, I did notice that the no_dup flag is defaulted to "False" in the pileup module. So, I would expect the duplicate reads to be included in pileup by default. Expected average depth at each of the base in the given interval is around 10,000 with duplicate reads. But pysamstats output average depth at these bases is around 400, which reflects the depth excluding duplicate reads.  Am I doing something wrong below? The bam file I'm using was processed through picards MarkDuplicates. Thanks for your help.

From command line:
`pysamstats --type variation_strand --no-dup --truncate --max-depth=80000 --chromosome 1 --start 11184502 --end 11184622 sample.bam  --fasta Homo_sapiens_assembly19.fasta`

From python:
`pysamstats.load_variation_strand(bamfile,chrom='1',start=11184502,end=11184622,truncate=True,fafile=fasta,max_depth=80000)`

**Comment by alimanfoo** [comment]
Hi @andurill, there is some work in master done by @hardingnj that addresses this, some new options to control read filtering, I believe there will be a way to turn off all read filters. Are you able to install from github? Latest pysamstats in master needs to be built against pysam 0.14. Something like this should work:

```
cd
mkdir spike-pysamstats
cd spike-pysamstats/
mkvirtualenv spike-pysamstats --python=/usr/bin/python3.6
pip install pysam
pip install numpy tables
git clone git@github.com:alimanfoo/pysamstats.git
cd pysamstats/
sudo apt-get install zlib1g-dev
python setup.py install
pysamstats --help
pysamstats --type=coverage fixture/test.bam | head
pysamstats --type=coverage --stepper=nofilter fixture/test.bam | head
```

The option ``--stepper=nofilter`` should return absolutely all reads. 

The default is equivalent to ``--stepper=all`` which I believe will filter duplicate reads.

**Comment by andurill** [comment]
Apologies for forgetting to close this issue. Your suggestion worked! Thank you.

**Comment by alimanfoo** [comment]
No problem. Btw this is now available in the latest release (1.1.2).
